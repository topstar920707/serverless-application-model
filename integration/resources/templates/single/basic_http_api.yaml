Resources:
  MyApi:
    Type: AWS::Serverless::HttpApi
    Properties:
      DefinitionBody:
        info:
          version: '1.0'
          title:
            Ref: AWS::StackName
        openapi: 3.0.1
        paths: {}