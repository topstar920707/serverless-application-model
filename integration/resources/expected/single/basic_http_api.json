[
  { "LogicalResourceId":"MyApi", "ResourceType":"AWS::ApiGatewayV2::Api"},
  { "LogicalResourceId":"MyApiApiGatewayDefaultStage", "ResourceType":"AWS::ApiGatewayV2::Stage" }
]